packages/replay-next/components/sources/Source.tsx

```diff
@@ -1,6 +1,14 @@
 import { newSource as ProtocolSource } from "@replayio/protocol";
 import debounce from "lodash/debounce";
-import { MouseEvent, Suspense, useContext, useLayoutEffect, useRef, useState } from "react";
+import {
+  MouseEvent,
+  Suspense,
+  useContext,
```

Comment:
Looks like an unused `useEffect` import got added here?

Reply:
Whoops, yeah, I had tried that file first and then changed. Will revert.